ch_tools/chadmin/chadmin_cli.py

```diff
@@ -70,9 +71,9 @@
     "--setting",
     "settings",
     multiple=True,
-    type=(str, str),
+    type=(str, YamlParamType()),
```

Comment:
Replaced `str` with `YamlParamType()` to be able to override the whole setting sections:

```
chadmin --setting cloud '{"metadata_service_endpint": "http://169.254.169.254"}' ...
```

The following syntax is supported as well:

```
chadmin --setting cloud.metadata_service_endpint "http://169.254.169.254" ...
```